definitions/consented/json/AuthorisationCaseField.json

```diff
@@ -4619,5 +4619,47 @@
     "CaseFieldID": "childrenInfo",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRUD"
+  },
```

Comment:
@johnoneill46  are you going to merge the latest master into this branch or shall I do it?

Should this field be in prod or nonprod?